shell/src/main/java/alluxio/shell/command/CheckConsistencyCommand.java

```diff
@@ -61,6 +61,16 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
     return 0;
   }
 
+  /**
+   * Checks the inconsistent files and directories which exist in Alluxio but don't exist in the
+   * under filesystem under a specified persisted path, prints the results and removes them if
```

Comment:
s/prints the ..../repairs the inconsistent paths by deleting them if repairConsistency is true